indico/modules/users/__init__.py

```diff
@@ -33,6 +33,7 @@
     'show_past_events': False,
     'name_format': NameFormat.first_last,
     'use_previewer_pdf': True,
+    'add_ical_alerts': True,
```

Comment:
Thinking about this again the empty string vs 0 is a bit of mess. Maybe better to use `0` for disabled? Or do you think people want a reminder *right* when the event starts? `-1` would work as well in that case, but otherwise I'd prefer `0`

Reply:
I think people will want a reminder when the event starts (if this proposal would support two alarms, I'd personally set it to -15 mins and then another at 0 mins). I can change the logic to be -1 instead of blank string as you propose if you'd like.

Reply:
the thing is that defaulting to enabled would affect all existing users and likely annoy a good chunk of them.

if 0 min should be possible, i suggest we go for two options: a bool to enable it, and a number to set the duration in minutes (using the HiddenUnless validator to hide it unless alarms are enabled in the form)

Reply:
It appears to work with -1 as the default in the new revision